Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -877,11 +880,34 @@ def config_search_request(
                 "sort": [{"direction": sort_direction, "field": sort_field}],
                 "timeRange": time_range,
                 "withResourceJson": include_resource_json,
-                "heuristicSearch": "true",
+                "heuristicSearch": heuristic_search,
             }
         )
 
-        return self._http_request("POST", "search/config", json_data=data)
+        first_page_response = self._http_request("POST", "search/config", json_data=data)
+        first_page_data = first_page_response.get("data", {})
+
+        items = first_page_data.get("items", [])
+        total_items.extend(items)
+        demisto.debug(f"POST search/config - {len(items)} items were fetched from page {page_count}")
+
+        page_limit = limit - len(items) if limit is not None else 0
+        next_page_token = first_page_data.get("nextPageToken")
+
+        while next_page_token and page_limit:
```

Comment:
The method config_search_request previously returned the full API response dict with structure {"data": {"items": [...], "totalRows": X}}. Now it returns only a list of items, and we are probably missing the totalRows. Are we ok with that?